test/unit/uk/gov/hmrc/tariffclassificationfrontend/connector/FileStoreConnectorSpec.scala

```diff
@@ -190,9 +186,12 @@ class FileStoreConnectorSpec extends UnitSpec with WiremockTestServer with Mocki
     val file = FileUpload(TemporaryFile("example-file.txt"), "file.txt", "text/plain")
     val result = await(connector.upload(file))
 
-    verify(postRequestedFor(urlEqualTo("/file"))
-      .withRequestBody(containing("file"))
-      .withRequestBody(containing("publish"))
+    verify(
+      postRequestedFor(
+        urlEqualTo("/file"))
+        .withHeader("X-Api-Token", equalTo(fakeAuthToken))
```

Comment:
In this case we do not use `http-verbs` - I had to use a different instance of `AppConfig` 
Let me know if you have a better idea here

Reply:
Thats fine, this connector method is an exception to the others as it uses WS